lib/internal/errors.js

```diff
@@ -960,7 +970,6 @@ E('ERR_ZLIB_INITIALIZATION_FAILED', 'Initialization failed', Error);
 
 function invalidArgType(name, expected, actual) {
   internalAssert(typeof name === 'string');
-  internalAssert(arguments.length === 3);
```

Comment:
Shouldn't we keep this check? Without this, the error can be implemented without passing in actual too, so no matter what the user passes as the actual argument, the error message will say `undefined`

Reply:
I just moved the check so it will be executed for all errors. That is actually what this PR is all about.